lib/bundler/plugin/events.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Bundler
+  module Plugin
+    module Events
+      def self.define(const, event)
```

Comment:
consider adding a yard macro to this, so we get docs for all of the constants?

Reply:
Opted to use `@parse` syntax below. I couldnt figure out macros >.<